app/Http/Transformers/SignupTransformer.php

```diff
@@ -67,4 +77,18 @@ public function includeEvents(Signup $signup)
 
         return $this->collection($event, new EventTransformer);
     }
+
+    /**
+     * Include the user data (optionally)
+     *
+     * @param \Rogue\Models\Signup $signup
+     * @return \League\Fractal\Resource\Item
+     */
+    public function includeUser(Signup $signup)
+    {
+        $registrar = new Registrar();
```

Comment:
We should try to make sure all class dependencies are either fetched using the `app()` helper or injected via the constructor so that Laravel can handle resolving dependencies for us. That will allow us to easily mock this class in a test case if we ever wanted to.